frontend/src/metabase/admin/databases/components/DatabaseConnectionInfoSection/DatabaseConnectionInfoSection.tsx

```diff
@@ -43,7 +42,7 @@ export const DatabaseConnectionInfoSection = ({
   };
 
   const handleDismissSyncSpinner = useCallback(
-    () => dismissSyncSpinner(database.id),
+    () => dismissSyncSpinner(database.id).unwrap(),
```

Comment:
Do we need to `unwrap` if we don't need the result for anything?

Reply:
The old function could throw, so I'm just mimicking that here.